packages/jellyfish-api-core/__tests__/category/wallet/listWallets.test.ts

```diff
@@ -0,0 +1,38 @@
+import { MasterNodeRegTestContainer, RegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Wallet without masternode', () => {
+  const container = new RegTestContainer()
+  const client = new ContainerAdapterClient(container)
```

Comment:
use `testing` instead of this.
https://github.com/DeFiCh/jellyfish/pull/957#discussion_r780976313

Reply:
Currently testing does not support `RegTestContainer`, only `MasterNodeRegTestContainer`. Should we be testing with both Master and non Master nodes or just Master node available to `@jellyfish/testing` ?

Reply:
hmm, that's true. you can keep the non masternode test as it is. trying to minimize the changeset when we move the container creation inside `Testing`
https://github.com/DeFiCh/jellyfish/issues/965 will track this.

Reply:
I have a local partial version of RegTestContainer addition to testing, will put up a draft later this week.